src/test/regress/expected/other_databases.out

```diff
@@ -0,0 +1,88 @@
+CREATE SCHEMA other_databases;
```

Comment:
As discussed over chat. We should add mx tests for this, i.e. CREATE/DROP USER should work from non citus main databases on workers too.